cirq-core/cirq/optimizers/align_left_test.py

```diff
@@ -44,3 +44,11 @@ def test_align_left():
             ]
         ),
     )
+
+
+def test_classical_control():
+    q0, q1 = cirq.LineQubit.range(2)
+    circuit = cirq.Circuit(
+        cirq.H(q0), cirq.measure(q0, key='m'), cirq.X(q1).with_classical_controls('m')
+    )
+    assert_optimizes(before=circuit, after=circuit.copy())
```

Comment:
Can you add these tests to https://github.com/quantumlib/Cirq/blob/master/cirq-core/cirq/transformers/align_test.py instead? `AlignLeft` / `AlignRight` are now deprecated after https://github.com/quantumlib/Cirq/pull/4891